app/models/validations/financial_validations.rb

```diff
@@ -24,11 +24,11 @@ def validate_net_income_uc_proportion(record)
   def validate_net_income(record)
     if record.ecstat1 && record.weekly_net_income
       if record.weekly_net_income > record.applicable_income_range.hard_max
-        record.errors.add :earnings, I18n.t("validations.financial.earnings.over_hard_max", hard_max: record.applicable_income_range.hard_max)
+        record.errors.add :earnings, :over_hard_min, message: I18n.t("validations.financial.earnings.over_hard_max", hard_max: record.applicable_income_range.hard_max)
```

Comment:
Guessing that's supposed to be over_hard_**max**?

Reply:
Great spot. Fixed!